### PR TITLE
3.17.0-Preview: Adds new preview release and updates changelog

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

This adds the changelog notes and updates the preview version to 3.17.0-preview

Changes between 3.15.0-preview and 3.17.0-preview

image

image

image

### CODE REVIEW

**Review by ealsur — APPROVED** [review]

**Comment by dpiessens** [comment]
Hello, it looks like you're creating this PR without the necessary working being completed. This PR #1526 was promised via support ticket 120122225009048 to be included in this release. Please complete this PR before releasing this as promised.

**Comment by ealsur** [comment]
@dpiessens that PR was closed based on inactivity, there was no activity since May 2020, you could contact the author to follow up but that work was not slated on this release.

**Comment by dpiessens** [comment]
> @dpiessens that PR was closed based on inactivity, there was no activity since May 2020, you could contact the author to follow up but that work was not slated on this release.

OK but it's linked to issue #993 which was reopened and assigned to someone on your team. Additionally we've been working with MS support who told us the issue would be done for 3.16, then that it didn't make that release and was slated for 3.17? So, either they were being dishonest, or your team was giving them false information. 

I can pick up this PR and make the remaining changes. Given the combination of how minor the change it, how it accounts for a significant gap in the C# API, and our willingness to contribute to something that was done, I'd ask you hold 3.17 until this can be incorporated so we don't need to wait another 8 weeks before consuming it.

**Review by kirankumarkolli — DISMISSED** [review]

File: changelog.md [hunk 1]
```diff
@@ -3,6 +3,21 @@ Preview features are treated as a separate branch and will not be included in th
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### <a name="3.17.0-Preview"/> [3.17.0-Preview](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.17.0-Preview) - 2021-02-08
+
+#### Added
+- [#2122](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2122) (Preview) Change Feed: Adds Full Fidelity support
+- [#2145](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2145) EnableContentResponseOnWrite: Adds client level support via CosmosClientOptions and CosmosClientBuilder 
+- [#2166](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2166) QueryRequestOption: Adds optimization to avoid duplicating QueryRequestOption
+- [#2206](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2206) LINQ : Adds User Defined Function Translation Support. (Thanks to dpiessens)
```
Comment (j82w) [inline changelog.md:12]: UDF support

**Review by ealsur — DISMISSED** [review]

**Review by kirankumarkolli — APPROVED** [review]

**Review by kirankumarkolli — APPROVED** [review]

**Pull request merged**